relayer/pkg/chainlink/txm/txm_test.go

```diff
@@ -75,6 +89,7 @@ func TestTxm(t *testing.T) {
 		}
 	}
 	time.Sleep(30 * time.Second)
+	wg.Wait()
```

Comment:
nit/ why sleep 30s if we are just going to `Wait()` anyways? doesn't that just artificially extend the length of the test? Can we drop the time.Sleep?

Reply:
if the `time.Sleep` is dropped then the txm would be closed right after the `wait` is fulfilled...but there are still txs in the queue at that point. so there would have to be a sleep afterward to make sure the the txm is done.

it's not ideal, but that `wait()` should immediately pass once the sleep time is done

Reply:
Ah, OK, two different things to wait on :up: